tests/parsers/test.manifestjson.js

```diff
@@ -26,8 +26,9 @@ describe('ManifestJSONParser manifestVersion', function() {
     var json = validManifestJSON({manifest_version: '1'});
     var manifestJSONParser = new ManifestJSONParser(json,
                                                     addonLinter.collector);
-    var result = manifestJSONParser.getMetadata();
-    assert.equal(result.isValid, false);
+    assert.equal(manifestJSONParser.isValid, false);
+
+    manifestJSONParser.getMetadata();
```

Comment:
I can't spot any side effects that would happen from calling this so it looks unnecessary.